iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -141,9 +141,10 @@ public void commitTask(TaskAttemptContext originalContext) throws IOException {
     TaskAttemptID attemptID = context.getTaskAttemptID();
     JobConf jobConf = context.getJobConf();
     Set<Path> mergedPaths = getCombinedLocations(jobConf);
+    String outputId = jobConf.get(HiveIcebergOutputFormat.ICEBERG_OUTPUT_ID);
```

Comment:
`Reason: There can be more than one OutputCommitter per vertex. If a job writes to 2 Iceberg tables in the same vertex, there can be 2 commit tasks happening in same vertex.`

can we use separate tasks for multiple outputs? cc @abstractdog